<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advance Setting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>29</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QLabel" name="label_adFindSurface">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>421</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Step 2 Find Surface</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_adminsurf">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_adminsurf">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>only useful if Mode is not &quot;simple&quot;</string>
     </property>
     <property name="text">
      <string>Min Surf</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_adncpu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>just use 1 here. &gt;1 may be slower in most cases...</string>
     </property>
     <property name="text">
      <string>CPUs</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_adncpu">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QLabel" name="label_adExtract_Surface">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>421</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Step 3 Extract Surface</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>make flattened tomo bigger (&gt;0) or smaller (&lt;0)</string>
     </property>
     <property name="text">
      <string>Expand ratio</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_adexpandratio">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <double>-0.300000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_ad_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>printtime</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_adprinttime">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Advance_Setting">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Advance Setting</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>361</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QLabel" name="label_adOther">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>421</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Other</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_adshow3d_bin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>bigger bin means less triangles in &quot;3D&quot; mode, but faster.</string>
     </property>
     <property name="text">
      <string>show3d_bin</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_adshow3d_bin">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_pointsin3d">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>79</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_pointsin3d">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>can draw picked particles as spheres in &quot;3D&quot; mode.</string>
     </property>
     <property name="text">
      <string>show points in 3D</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>377</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
